### PR TITLE
docs: update arbos info

### PR DESCRIPTION
**Pull request by leeederek**
- adds a small note on our general recommendation for upgrading Nitro node software
- Updates the arbOS 20 description with the batch poster manager role and sequencerinbox finality fix as outlined [here on snapshot](https://snapshot.org/#/arbitrumfoundation.eth/proposal/0xbecc45a6beb55a708e25195f34db2f5ff757e0da5ff55171ca731b39428e27c3)
- Adds a link to Ed's medium blog post on the topic of EIP4844 impacts on L2 users
- Add more descriptive reference link headers

### CODE REVIEW

File: arbitrum-docs/node-running/reference/software-releases.md [hunk 1]
```diff
@@ -13,7 +13,13 @@ Arbitrum chains are powered by Arbitrum nodes running the Nitro software stack.
 
 Although Nitro is updated regularly, only a subset of Nitro releases carry ArbOS upgrades. These releases are significant because ArbOS upgrades are Arbitrum's equivalent to a hard fork - they can alter a node's ability to produce valid Arbitrum blocks. This is why validator nodes supporting a public Arbitrum chain (One, Nova) **must update Nitro** whenever a new ArbOS version is released.
 
-Note that every Nitro release is backwards compatible. In other words, the latest version of Nitro will support all previous ArbOS releases. This means that your validator's Nitro version must be greater than or equal to the version that includes the latest ArbOS upgrade.
+Note that every Nitro release is backwards compatible. In other words, the latest version of Nitro will support all previous ArbOS releases. This means that your validator's Nitro version must be greater than or equal to the version that includes the latest ArbOS upgrade.
```
Comment (symbolpunk) [inline arbitrum-docs/node-running/reference/software-releases.md:16]: If aligned on the above edit, we can use it here, too. Duplicated copy can be tough to maintain; partials solve that problem - I'd recommend using one here.

File: arbitrum-docs/node-running/reference/software-releases.md [hunk 2]
```diff
@@ -25,6 +31,12 @@ Beginning with ArbOS 20, ArbOS releases use the name of planetary moons in our s
 
 The number used to denote each upgrade will increment by 10, starting from ArbOS 20 (i.e. the next ArbOS upgrade after ArbOS 20 will be ArbOS 30). This was done because there are teams who have customized their Orbit chain's [behavior](../../launch-orbit-chain/how-tos/customize-stf.mdx) or [precompiles](../../launch-orbit-chain/how-tos/customize-precompile.mdx) and who may wish to use ArbOS's naming schema between official ArbOS version bumps (e.g. ArbOS 12 could be the name of a customized version of ArbOS for a project's L3 Orbit chain).
 
+### Stay up to date with the latest information on the status of network upgrades
```
Comment (symbolpunk) [inline arbitrum-docs/node-running/reference/software-releases.md:34]:
```suggestion
### Stay up to date
```

Repeated below; tight headers are generally preferred for skimmability

**Review by symbolpunk — APPROVED** [review]
Nits noted as suggested edits 👍

File: arbitrum-docs/node-running/reference/software-releases.md (hunk 1 above)
Comment (leeederek) [reply arbitrum-docs/node-running/reference/software-releases.md:16]: Makes sense, i've added a partial and referenced it in both places!

**Pull request merged**